msu/systems/mod_settings/abstract_setting.nut

```diff
@@ -64,28 +79,33 @@
 		this.set(_value, _updateJS, _updatePersistence, _updateCallback, _force);
 	}
 
-	function set( _value, _updateJS = true, _updatePersistence = true, _updateCallback = true, _force = false)
+	function set( _newValue, _updateJS = true, _updatePersistence = true, _updateBeforeChangeCallback = true, _force = false, _updateAfterChangeCallback = true)
```

Comment:
Oh boy this is ugly. Perhaps deprecate `set` and write a new `setValue` function with a better argument order?

Reply:
I think we go with the pain for now and add a new function that accepts kwargs

Reply:
I mean eventually we have to do that anyway IF we do kwargs i.e. we'll have to add a new function. So this is fine.